### PR TITLE
[macros, lex, over] Dissolve bnfkeytab environments. Use a fixed-width (\fw{...}) code field instead.

### PR DESCRIPTION
**Pull request by tkoeppe**
Before/after:

image

image

### CODE REVIEW

File: source/lex.tex [hunk 1]
```diff
@@ -792,18 +792,20 @@
...
-\><: \>:> \><\% \>\%> \>\%: \>\%:\%: \>; \>: \>.{..}\br
-\>new \>delete \>? \>:: \>. \>.* \>-> \>->* \>\~\br
-\>! \>+ \>- \>* \>/ \>\% \>\caret \>\& \>|\br
-\>= \>+= \>-= \>*= \>/= \>\%= \>\caret= \>\&= \>|=\br
-\>== \>!= \>< \>> \><= \>>= \><=> \>\&\& \>||\br
-\><< \>>> \><<= \>>>= \>++ \>-{-} \>,\br
-\>and \>or \>xor \>not \>bitand \>bitor \>compl\br
-\>and_eq \>or_eq \>xor_eq \>not_eq
-\end{bnfkeywordtab}
+\fw{\{}\fw{\}}\fw{[}\fw{]}\fw{\#}\fw{\#\#}\fw{(}\fw{)}\br
```
Comment (godbyk) [inline source/lex.tex]: What about using the `\terminal` macro like in [[lex.name] above](https://github.com/cplusplus/draft/pull/2022/files#diff-a803830b6534424eb14ecf959b859c07R531)? It'd certainly be easier to read the source code that way.

Comment (tkoeppe) [reply source/lex.tex]: The spaces collapse, so the output would look very crowded.

Comment (godbyk) [reply source/lex.tex]: Oh. Throw in an `\obeyspaces` and then the spaces won't collapse.

Comment (tkoeppe) [reply source/lex.tex]: Hah, you win this one. Thanks!

**Comment by tkoeppe** [comment]
I went with the `\terminal` version suggested by @godbyk. Thank you!

**Pull request closed without merging**